src/plugins/memdump/private.h

```diff
@@ -108,6 +108,8 @@
 #include <vector>
 #include <list>
 
+#define THREAD_WOW64_CONTEXT 29
```

Comment:
This needs to be documented in some way. Enum from rekall profile would be nice, else add good bit of docs here.